### ISSUE TITLE
Startup path or different home directory for syncthing

### ISSUE THREAD

**Issue by wikibit** [syncthing/syncthing-gtk#305]
Hi,

is it possible to start syncthing-gtk using a different "home" directory than the standard one?

e.g. on windows (10) 
I've all my syncthing setups running from C:\system\syncthing
and the config and data is in C:\system\syncthing\data
Therefor I also start the daemon with "-home=c:\system\syncthing\data"

during the setup of syncthing-gtk, it will note that it did not find any syncthing setup, if there is, I should check ~/.config/syncthing folder, but that does not exist on windows of course

Basically, what I would like, is to start the UI so that it uses the syncthing setup in my existing directories

is that possible?

Thanks

**Comment by kozec** [comment]
It should be possible by setting HOME environment variable for ST-GTK, but `syncthing/config.xml` is kinda hardcoded right now. Where exactly is your `config.xml` located?

**Comment by wikibit** [comment]
Hi,

 

My config.xml is currently syncthing/data/config.xml

**Comment by kozec** [comment]
> My config.xml is currently syncthing/data/config.xml

Ok, so that's not solvable just by changing $HOME :(

Adding -home argument to ST-GTK as well is probably least-crazy solution to this.

**Comment by kozec** [comment]
In next release, running `syncthing-gtk --home C:\\system\\syncthing\\data` should be equivalent to running `syncthing -home C:\\system\\syncthing\\data`

**Comment by wikibit** [comment]
Hi,

 

OK, I will try that as soon as I can install the next release

 

L.
